_benchmark/user-guide/contributing-workloads.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: Contributing custom workloads
+nav_order: 11
+parent: User guide
+---
+
+# Contributing custom workloads
+
+After [creating a custom workload]({{site.url}}{{site.baseurl}}/benchmark/creating-custom-workloads/) you can contribute your workload for other OpenSearch Benchmark users to use. If you plan on contibuting a workload, use the following guidelines.
+
+## README and license
+
+Each workload must include a detailed `README.MD` file, which should include the following:  
+
+- The purpose of the workload and what users should use the workload. When creating a description, think about what specific use case your workload address that other workloads included in OpenSearch Benchmark do not.
+- An overview of the dataset used by the workload. Make sure that any data included in the dataset does not contain any proprietary data or Personal Identifiable Information (PII). 
+- An example document from the data set that gives users an idea of how the data looks like.
+- The parameters supported by the workload.
+- A list of default test procedures included in the workload and other test procedures the workload can run.
+- A sample of the output the workload produces after a test is run.
+- A copy of the license that gives the user and OpenSearch Benchmark permission to use the dataset.
+
+
+For an example workload README, see the `http_logs` [README](https://github.com/opensearch-project/opensearch-benchmark-workloads/blob/main/http_logs/README.md).
+
+## Structure
+
+Make sure that your workload includes the following files. For more details on the contents of each file, see [Anatomy of a workload]({{site.url}}{{site.baseurl}}/benchmark/user-guide/understanding-workloads/anatomy-of-a-workload/).
+
+- `workload.json`
+- `index.json`
+- `files.txt`
+- `test_procedures/default.json`
+- `operations/default.json` 
+
+Both `default.json` file names can be customized to have a descriptive name.
+
+The workload can also include an optional `workload.py` file to add more dynamic functionality to the workload.
+
+## Testing the workload
+
+All workloads contributed to OpenSearch Benchmark must fulfil the following testing requirements: 
+
+- All tests run to explore the workload and produce examples should be run on OpenSearch clusters.
```

Comment:
- All tests run to explore the workload and produce examples should be targeted to OpenSearch clusters.